tests/conftest_modules/test_configuration.py

```diff
@@ -65,6 +65,11 @@
             ]
         },
         "include_outliers": True,
+        "preserve_case": {
+            "database": True,
+            "schema": True,
+            "relation": True
+        },
```

Comment:
For the existing tests, we should leave this as the default or explicitly set to all `False`.

Additional tests for the preserving case functionality can pick up this config and modify the `"preserve_case"` flag to verify the added behavior.